scripts/src/main/resources/scripts/setup-kubernetes

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+kubernetes_check=$(kubectl version --client)
+if  [[ ! $kubernetes_check == *"Client Version"* ]]; then
+    echo Installing Kubectl...
+    curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg ^| sudo apt-key add -
+    sudo touch /etc/apt/sources.list.d/kubernetes.list
+    echo "deb http://apt.kubernetes.io/ kubernetes-xenial main" ^| sudo tee -a /etc/apt/sources.list.d/kubernetes.list
+    sudo apt-get update
+    sudo apt-get install -y kubectl
+    echo Kubernetes installed successfully.
+else 
+    echo Kubernetes already installed. 
+fi
```

Comment:
please also echo the kubernetes version information after fi

Reply:
Done